deltachat-ios/Controller/BlockedContactsViewController.swift

```diff
@@ -43,7 +43,7 @@ class BlockedContactsViewController: GroupMembersViewController, GroupMemberSele
     func selected(contactId: Int, selected: Bool) {
         if !selected {
             let dcContact = DcContact(id: contactId)
-            let title = dcContact.displayName.isEmpty ? dcContact.email : dcContact.displayName
+            let title = dcContact.displayName
```

Comment:
just came over that when grep'ing for displayName: when displayName is empty it is set to address by definition.